diff --git a/src/App.js b/src/App.js
index d7b8435..ebc2af5 100755
--- a/src/App.js
+++ b/src/App.js
@@ -5,7 +5,6 @@ import Spinner from './components/common/Spinner/Spinner';
 import Logout from './components/Accounts/Logout/Logout';
 import './App.scss';
 
-const DalalBull = lazy(() => import('./components/Dalalbull/DalalBull'));
 const Kryptos = lazy(() => import('./components/Kryptos/Kryptos'));
 const Login = lazy(() => import('./components/Accounts/Login/Login'));
 
@@ -16,7 +15,6 @@ const App = () => (
         <Route exact path="/" component={Home} />
         <Route path="/login" component={Login} />
         <Route path="/logout" component={Logout} />
-        <Route path="/dalalbull" component={DalalBull} />
         <Route path="/kryptos" component={Kryptos} />
       </Suspense>
     </div>
diff --git a/src/components/Home/Home.js b/src/components/Home/Home.js
index 0f25aa7..71b7080 100644
--- a/src/components/Home/Home.js
+++ b/src/components/Home/Home.js
@@ -41,9 +41,8 @@ const Home = () => {
                 name="DALALBULL"
                 rank={dalalbullRank}
                 logo={dalal}
-                type="ranked"
-                href="/Dalalbull"
-                status="PLAY NOW"
+                type="notRanked"
+                status="COMING SOON"
               />
             </div>
             <div className="col-lg-6 center-block text-center cell">
@@ -62,7 +61,7 @@ const Home = () => {
                 name="#INCLUDE"
                 logo={hash}
                 type="notRanked"
-                status="PLAY NOW"
+                status="COMING SOON"
               />
             </div>
             <div className="col-lg-6 center-block text-center cell">
@@ -70,7 +69,7 @@ const Home = () => {
                 name="CIRCUIMSTANCE"
                 logo={circ}
                 type="notRanked"
-                status="PLAY NOW"
+                status="COMING SOON"
               />
             </div>
           </div>
